### DISCUSSION TITLE
Purpose of virtual bones

### DISCUSSION THREAD

**Discussion by murataka** [Sixze/ALS-Refactored#262] [General]
Hello,

I just removed some virtual bones from live retargeted skeletal meshes (retarget pose from pose method in animbp) , and I see no difference. As I have no idea what the purpose of  those virtual bones  is  , I do not know what consequences that might cause. 

So , What is the main purpose of virtual bones ? Am I just ok if I delete them from those live retargeted meshes ? And also , if they have a good purpose , could you provide me a good tutorial , so we can master it ?

We have some explanation on https://docs.unrealengine.com/5.1/en-US/virtual-bones-in-unreal-engine/ , but I want to discuss this in the scope of ALS-Refactored implemented features. Also , do they have any affect , if without extra coding , like setting max possible distance between hands constraint ?

**Comment by oivio** [comment]
As I did seen some virtual bones has been used on Overlays for weapon handling like M4, Bow etc.
